src/test/java/AddNewContactTest.java

```diff
@@ -0,0 +1,44 @@
+import org.testng.annotations.Test;
+
+import static java.lang.Thread.sleep;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AddNewContactTest extends BaseTest {
+
+
+    @Test
+    public void successAddNewContact() throws InterruptedException {
+        LoginHelper loginHelper = new LoginHelper(app.driver);
+        loginHelper.loginTestHelper();
+        ContactsPage contactsPage = new ContactsPage(app.driver);
+        sleep(2000);
+        contactsPage.clickButtonAddNewContact();
+        assertTrue(contactsPage.modalWindowIsDisplayed());
+        AddContactDialog addContactDialog = new AddContactDialog(app.driver);
+        addContactDialog.enterFirstName("Maria");
+        addContactDialog.enterLastName("Marz");
+        addContactDialog.enterAbout("123");
+        addContactDialog.clickButtonSave();
+        ContactInfoPage contactInfoPage = new ContactInfoPage(app.driver);
+        sleep(2000);
+        assertEquals(contactInfoPage.firstNameIsCorrect(), "Maria");
+        assertEquals(contactInfoPage.lastNameIsCorrect(), "Marz");
+        assertEquals(contactInfoPage.descriptionIsCorrect(), "123");
+        contactInfoPage.clickButtonEdit();
+        contactInfoPage.editAllInputFields("Maria9", "Marz9", "1239");
+        contactInfoPage.clickButtonSave();
+        contactInfoPage.clickButtonContact();
+        contactsPage.clickSearchInputField();
+        contactsPage.enterNameInputSearchField("Maria9");
+        assertEquals(contactsPage.contactIsCorrect(), "MariaMaria9 MarzMarz9");
```

Comment:
I see that when editing contact data, new names were added, not replaced the old names.

Reply:
In addition, you added "Maria9" when editing, but you check with the name “MariaMaria9"

Reply:
Yes